exonum-light-client/src/main/java/com/exonum/client/response/TransactionResponse.java

```diff
@@ -73,4 +74,30 @@ public boolean isCommitted() {
     return status == COMMITTED;
   }
 
+  @Override
+  public String toString() {
+    return "TransactionResponse(status=" + this.getStatus()
+        + ", message=" + this.getMessage()
+        + (isCommitted() ? ", executionResult=" + this.getExecutionResult() : "")
+        + ", location=" + this.getLocation() + ")";
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    TransactionResponse that = (TransactionResponse) o;
+    return status == that.status
```

Comment:
How likely it is that someone (us included) will create objects with same message but different result/location (e.g., for test purposes)?